src/cmake/dependencies.cmake

```diff
@@ -7,10 +7,12 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}")
 
 # Find Corrade first so we can use CORRADE_TARGET_*
 if(NOT USE_SYSTEM_MAGNUM)
-  # These are enabled by default but we don't need them right now -- disabling
-  # for slightly faster builds. If you need any of these, simply delete a line.
-  set(WITH_INTERCONNECT OFF CACHE BOOL "" FORCE)
-  add_subdirectory("${DEPS_DIR}/corrade")
+    # These are enabled by default but we don't need them right now -- disabling
+    # for slightly faster builds. If you need any of these, simply delete a line.
+    set(WITH_INTERCONNECT
+        OFF
+        CACHE BOOL "" FORCE)
```

Comment:
I wonder what's the logic behind these aggressive line breaks for `set` arguments.

Reply:
@eundersander explained in the docs: https://cmake-format.readthedocs.io/en/latest/format-case_studies.html

Reply:
It looks like this style is to accommodate comments on each argument, which makes sense.

Reply:
I could also give it control over formatting the cmake comments, but have opted not to for the first run.